build-scripts/gulp/alerts.js

```diff
@@ -24,9 +24,9 @@ class VersionedItem {
       part = part.replace(/,/, "");
 
       if (part[0] === ">") {
-        this.min = part.substr(1);
+        this.introduced_in = part.substr(1);
```

Comment:
"Introduced in" is not a good name. The issue is not introduced in that version. That HA version is affected by it.